### PR TITLE
ONGR strict psr fixes

### LINKED ISSUE #113: Fix code style to meet ongr-strict-standard

### PR DESCRIPTION
**Pull request by martiis**
Closes #113 

Annotations will be fixed with #75

### CODE REVIEW

**Comment by martiis** [comment]
Random travis fails.

**Comment by martiis** [comment]
Failures should be fixed with https://github.com/ongr-io/ElasticsearchBundle/pull/129

File: DSL/Filter/AbstractFilter.php [hunk 1]
```diff
@@ -25,9 +25,9 @@
     protected $filters;
 
     /**
-     * @param array $boolParams Bool parameters.
+     * Constructor.
```
Comment (trylika) [inline DSL/Filter/AbstractFilter.php]: useless comment, description may be empty for constructor

File: DSL/Filter/IndicesFilter.php [hunk 2]
```diff
@@ -34,6 +34,8 @@ class IndicesFilter implements BuilderInterface
     private $noMatchFilter;
 
     /**
+     * Constructor.
```
Comment (trylika) [inline DSL/Filter/IndicesFilter.php]: useless comment, description may be empty for constructor

File: DSL/Highlight/Field.php [hunk 3]
```diff
@@ -23,63 +23,49 @@ class Field
...
-     * Highlight fields based on the source.
-     *
-     * @var bool
+     * @var bool Highlight fields based on the source.
      */
     protected $forceSource;
 
     /**
-     * @param string $name
+     * Constructor.
```
Comment (trylika) [inline DSL/Highlight/Field.php]: useless comment, description may be empty for constructor

File: DSL/Query/IndicesQuery.php [hunk 4]
```diff
@@ -34,6 +34,8 @@ class IndicesQuery implements BuilderInterface
     private $noMatchQuery;
 
     /**
+     * Constructor.
```
Comment (trylika) [inline DSL/Query/IndicesQuery.php]: useless comment, description may be empty for constructor

File: DSL/Sort/Sort.php [hunk 5]
```diff
@@ -19,13 +19,13 @@
...
     /**
-     * Filter for sorting.
-     *
-     * @var BuilderInterface
+     * @var BuilderInterface Filter for sorting.
      */
     private $nestedFilter;
 
     /**
+     * Constructor.
```
Comment (trylika) [inline DSL/Sort/Sort.php]: useless comment, description may be empty for constructor

File: DSL/Filter/AbstractFilter.php (hunk 1 above)
Comment (tautrimas) [inline DSL/Filter/AbstractFilter.php]: It's not strictly useless, but not necessary in these cases ;)

**Pull request merged**